src/test/java/seedu/address/logic/commands/ClientNoteAddCommandTest.java

```diff
@@ -53,7 +61,34 @@ public void execute_notDuplicateClientNote_successfullyAddsClientNote() {
         } catch (CommandException e) {
             fail();
         }
+    }
+
+    @Test
+    public void equals() {
+        Index idx = Index.fromOneBased(1);
+        Index idx2 = Index.fromOneBased(2);
+        Note clientNote1 = new Note(NOTE_CONTENT_2);
+        Note clientNote2 = new Note(NOTE_CONTENT_1);
```

Comment:
Is there a reason `clientNote1` is `NOTE_CONTENT_2`?

Reply:
My bad, fixed it